README.md

```diff
@@ -215,12 +219,41 @@ export SERVICE_TRIGGER_ADDR=`jq -r .deployedTo .docker/trigger.json`
 Deploy the compiled component with the contract information from the previous steps. Review the [makefile](./Makefile) for more details and configuration options.`TRIGGER_EVENT` is the event that the trigger contract emits and WAVS watches for. By altering `SERVICE_TRIGGER_ADDR` you can watch events for contracts others have deployed.
 
 ```bash docci-delay-per-cmd=2
-# Build your service JSON with optional overrides in the script
+# Build your service JSON
 COMPONENT_FILENAME=eth_price_oracle.wasm AGGREGATOR_URL=http://127.0.0.1:8001 sh ./script/build_service.sh
 
-# Deploy the service JSON to WAVS so it now watches and submits
-# the results based on the service json configuration.
-SERVICE_CONFIG_FILE=.docker/service.json make deploy-service
+# Deploy the service JSON to WAVS so it now watches and submits.
+#
+# If CREDENTIAL is not set then the default WAVS_CLI .env account will be used
+# You can `cast send ${WAVS_SERVICE_MANAGER} 'transferOwnership(address)'` to move it to another account.
+SERVICE_CONFIG_FILE=.docker/service.json CREDENTIAL=${DEPLOYER_PK} make deploy-service
+```
+
+
+## Register service specific operator
```

Comment:
This section is super ugly but a requirement right now for alpha.5. Can cleanup closer to 0.4.0